tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
except, out captures *already* have time information in them.  We *already* have an option to match exactly what the server does, which pulls the timing info from the recording and submits the frames as they came in.  We don't want the code here messing with that.